cypress-tests/cypress/support/commands.js

```diff
@@ -1250,7 +1250,9 @@ Cypress.Commands.add(
   "saveCardConfirmCallTest",
   (saveCardConfirmBody, req_data, res_data, globalState) => {
     const paymentIntentID = globalState.get("paymentID");
-    saveCardConfirmBody.card_cvc = req_data.payment_method_data.card.card_cvc;
+    if (req_data.setup_future_usage === "on_session"){
+      saveCardConfirmBody.card_cvc = req_data.payment_method_data.card.card_cvc;
```

Comment:
If you are removing card_cvc from the saveconfirmBody . How this check will work "saveCardConfirmBody.card_cvc"?

Reply:
saveCardConfirmBody.card_cvc will overwrite the fixture card_cvc